### DISCUSSION TITLE
Managed configuration client authencation not working EMM Tectoro

### DISCUSSION THREAD

**Discussion by manish-tectoro** [strongswan/strongswan#2287] [Q&A]
Hi community team,
we are facing an issue in implementing managed configuration on Android MDM, we're able to connect through personal android mobile device but when tried same configuration through managed config. VPN server deployed on AWS EC2.
![Managed config](https://github.com/strongswan/strongswan/assets/172471743/9dd1b28b-5bb6-4a44-bab2-a537712d59d1)
![Managed config1](https://github.com/strongswan/strongswan/assets/172471743/a99f3a11-ae9f-4e16-86c5-8ee718e63d44)
![Mobile config ](https://github.com/strongswan/strongswan/assets/172471743/443f21bc-4577-48db-8b98-a47bac9e033f)

**Comment by tobiasbrunner** [comment]
What exactly doesn't work? Log?

**Reply by manish-tectoro** [reply]
When we pushing user credential and server details, we're unable to get profile in application, but at the same time when we blocked profile import or certificate import for user that is working.

we didn't get any logs on mobile device.

**Reply by tobiasbrunner** [reply]
I just saw that you haven't configured a profile name. While it does say `(optional)`, that's not actually true for managed configurations. That label is taken from the GUI, where it actually is optional because it defaults to the server address there.  However, for managed profiles there is currently no such fallback.
